src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,15 +2,23 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
         //TODO: add corresponding log message about method login was called
         User user = findByLogin(login);
+        logger.info("Login method was called for User with login: {}", login);
         if (!user.getPassword().equals(password)) {
-            throw new AuthenticationException("Username or password are incorrect");
+            logger.error("WRONG PASSWORD! "
+                    + "AuthenticationException throws for User with login: {} ", login);
```

Comment:
According to the checklist, we should remove all TODO comments before submitting the solution. The message 'WRONG PASSWORD!' seems like a placeholder and might not be suitable for production logs. Instead, use a more formal log message.